CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ AppEngine version, listed here to ease deployment and troubleshooting.
 
 ## Next Release (replace with git tag when deployed)
 
+## `20210726t065700-all`
```

Comment:
I think we usually keep the one that is going to get traffic. If it doesn't get traffic, let's remove it?